wcfsetup/setup/db/install.sql

```diff
@@ -1446,6 +1446,20 @@ CREATE TABLE wcf1_trophy(
 	KEY(categoryID)
 );
 
+DROP TABLE IF EXISTS wcf1_unfurl_url;
+CREATE TABLE wcf1_unfurl_url (
+	urlID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	url TEXT NOT NULL,
+	urlHash VARCHAR(40) NOT NULL UNIQUE KEY (urlHash),
+	title VARCHAR(255) NOT NULL DEFAULT '',
+	description TEXT,
+	imageUrl TEXT NOT NULL DEFAULT '',
+	imageType VARCHAR(255) NOT NULL DEFAULT 'NOIMAGE',
+	imageHash VARCHAR(45) NOT NULL DEFAULT '',
+	status VARCHAR(255) NOT NULL DEFAULT 'PENDING',
+	lastFetch INT(10) NOT NULL DEFAULT 0
```

Comment:
```suggestion
	lastFetch INT(10) NOT NULL
```
The default just doesn't make sense. It should be explicitly provided.

Reply:
No, because, if the URL is not fetched yet, it is `0` (and no, we don't use `null` for that at any other place). And every other column has a default value, too.